packages/klighd-core/src/views-common.ts

```diff
@@ -145,7 +145,7 @@ export function textDecorationStyleText(underline: KTextUnderline): 'solid' | 'd
     }
 }
 
-export function textDecorationColor(underline: KTextUnderline): string | undefined {
```

Comment:
This parameter is needed, if the implementation for this would be completed, if this throws some eslint error, have it just ignore that.